src/NHibernate/NHibernate.csproj

```diff
@@ -12,7 +12,6 @@
 
     <PackageDescription>NHibernate is a mature, open source object-relational mapper for the .NET framework. It is actively developed, fully featured and used in thousands of successful projects.</PackageDescription>
     <PackageTags>ORM; O/RM; DataBase; DAL; ObjectRelationalMapping; NHibernate; ADO.Net; Core</PackageTags>
-    <LangVersion>7.3</LangVersion>
```

Comment:
Need to explicitly specify language version. I propose use 10.

Reply:
Maybe should be added to NHibernate.props instead

Reply:
latestMajor (default) | The compiler accepts syntax from the latest released major version of the compiler.

I don't understand why it doesn't work by default. Shouldn't latestMajor be 10.0 too?

Reply:
> I don't understand why it doesn't work by default. Shouldn't latestMajor be 10.0 too?

It is tied to the target framework somehow.

Reply:
That explains it, though it's not what description says... Some strange compilations issues with 10.0  Let's start with 9.0 for now..